tests/integration/test_ec2.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import botocore.session
+
+from awscli.testutils import unittest, aws
+
+
+class TestDescribeInstances(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Create an instance in the scope of a class so that only one needs to
+        # be created during the full test run.
+        cls.session = botocore.session.get_session()
+        cls.region = 'us-west-2'
+        cls.client = cls.session.create_client('ec2', cls.region)
+        response = cls.client.run_instances(
```

Comment:
That's a good point. The point of these tests is to verify the service's parameter handling, and EC2 conveniently provides `DryRun`. That should be sufficient.